api/src/udfs/llm_summarizer_udf.py

```diff
@@ -57,7 +57,7 @@ def content_summarizer(control_message: "IngestControlMessage") -> "IngestContro
     logger.info("UDF: Starting LLM content summarization")
 
     api_key = os.getenv("NVIDIA_API_KEY")
-    model_name = os.getenv("LLM_SUMMARIZATION_MODEL", "nvidia/llama-3.1-nemotron-70b-instruct")
+    model_name = "moonshotai/kimi-k2-instruct-0905"
```

Comment:
Just wondering if this was left from a debugging session or if you intentionally meant to remove support for reading the env var.